guides/common/modules/snip_prerequisite-configured-smart-proxy-registration-provisioning.adoc

```diff
@@ -0,0 +1,9 @@
+* Optional: In the scenario in which you want to use {SmartProxyServers} instead of the {ProjectServer}, ensure that you have configured your {SmartProxies} accordingly.
```

Comment:
```suggestion
* Optional: If you want to use {SmartProxyServers} instead of the {ProjectServer}, ensure that you have configured your {SmartProxyServers} accordingly.
```

Reply:
I'm not sure if "Optional" is appropriate. It is rather "Conditional" than "Optional", but mandatory if you use SmartProxies. Would you agree with removing "Optional"?

Reply:
Yes, I'd remove optional if it's in fact conditional.

I only think `Optional: ...` is appropriate for adding descriptions etc. If something has to be clicked/checked/done under certain conditions, we should IMHO always go with `If X, then Y.`.